### PR TITLE
Fix GitHub Codespaces

### PR DESCRIPTION
**Pull request by ethax-ross**
The main issues fixed in this PR are:

- Expose the Postgres port from the container to the host
- Set Postgres credentials on Dockerfile
- Create `vscode` role/user in Postgres on boot
- Start `rails server` automatically
- Disable CSRF checks on Codespaces (as its failing but I'm not sure why)

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -78,4 +78,10 @@
     api_enabled: true,
     migration_enabled: true,
   }
+
+  # Disable origin check for Cross-Site Request Forgery (CSRF) protection for codespaces.
```
Comment (peteryates) [inline config/environments/development.rb:82]: Does this happen when the CodeSpace is running in a browser or locally via VS Code? Or both? 🤔

Comment (ethax-ross) [reply config/environments/development.rb:82]: I tested it running in a browser as I figured that's how the content editors are going to use it... its an odd one but there's a few similar reports kicking around in various GitHub issues about it

**Review by peteryates — APPROVED** [review]
Nice, this looks great!

File: config/environments/development.rb (hunk 1 above)
Comment (peteryates) [reply config/environments/development.rb:82]: Yeah, I've seen it in the past but from memory the process is smoother when running from VS Code locally. But the fix  here looks like it'll do the trick nicely 👌🏽

**Pull request merged**